extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -431,22 +434,50 @@
 			}
 		};
 
-		if (_t.updateInterval === 0) {
-			return;
-		}
-		// Limit the refresh interval to 100 ms
-		if (_t.updateInterval < 100) {
-			_t.updateInterval = 100;
-		}
+		_t.setVisible = function(state) {
+			if (state) {
+				_t.formRow.classList.remove(o.formRowHidden);
+				_t.activateRefresh();
+			} else {
+				_t.formRow.classList.add(o.formRowHidden);
+				_t.disactivateRefresh();
+			}
+
+			_t.visible = state;
+		};
+
+		_t.disactivateRefresh = function() {
+			if (interval !== undefined) {
+				clearInterval(interval);
+				interval = undefined;
+			}
+		};
+
+		_t.activateRefresh = function() {
+			_t.disactivateRefresh();
+
+			if (_t.updateInterval === 0) {
+				return;
+			}
+			// Limit the refresh interval to 100 ms
+			if (_t.updateInterval < 100) {
+				_t.updateInterval = 100;
+			}
 
-		var
 			interval = setInterval(function() {
 				if (_t.image.clientWidth === 0) {
 					clearInterval(interval);
 					return;
 				}
-				_t.image.setAttribute("src", _t.url + "&t=" + Date.now());
+				if (_t.image.getAttribute("src").startsWith(_t.url)) {
```

Comment:
Couldn't figure out what this is for.

Reply:
It is to avoid a refresh of the image using the proxied URL when the image was set with a raw data for example.

Reply:
Maybe I should rather set a variable to store the information whether we should refresh or not.